src/ControlSystem/ControlErrors/Translation.hpp

```diff
@@ -42,19 +42,19 @@ namespace ControlErrors {
  * \details Computes the error in how much the system has translated by using
  * Eq. (42) from \cite Ossokine2013zga. The equation is
  *
- * \f[ \left(0, \delta\vec{T}\right) = a\mathbf{q}\left(\mathbf{x}_B -
- * \mathbf{c}_B - \mathbf{\delta q}\wedge\mathbf{c}_B - \frac{\delta
- * a}{a}\mathbf{c}_B \right)\mathbf{q}^*
+ * \f[ \left(0, \delta\vec{T}\right) = a\mathbf{q}\left(\mathbf{x}_A -
+ * \mathbf{c}_A - \mathbf{\delta q}\wedge\mathbf{c}_A - \frac{\delta
+ * a}{a}\mathbf{c}_A \right)\mathbf{q}^*
  * \f]
  *
- * where object B is located on the positive x-axis, bold face letters are
+ * where object A is located on the positive x-axis, bold face letters are
```

Comment:
do we want to say "in the grid frame" here?

Reply:
Yes I'll add that. It'll have to be changed once we move to the distorted frame, but so will everything else so that's ok.